pkgs/os-specific/linux/kernel/xanmod-kernels.nix

```
@@ -2,15 +2,15 @@
 
 let
   stableVariant = {
-    version = "5.15.40";
+    version = "5.15.43";
     suffix = "xanmod1";
-    hash = "sha256-P9pRPfzBVQNhWp6ulDiBgV/ki2C0CTbIRV6KRhUD+vQ=";
+    hash = "sha256-MeH9RUPDiuN22eAZ18v+N3aIT18dQ3FnTkcQV0MjB4k=";
   };
 
   edgeVariant = {
-    version = "5.17.8";
+    version = "5.18.1";
     suffix = "xanmod1";
-    hash = "sha256-Q8YmBeocolr6Ds2inwV299Td6/zE8RTA1SWve5ZykAs=";
+    hash = "sha256-dqvB4F2S7cklSJ7XTUNvWVKTsZGLevOXME5lvhmfyis=";
   };
 
   xanmodKernelFor = { version, suffix, hash }: buildLinux (args // rec {
```

```suggestion
  xanmodKernelFor = { version, suffix, hash }: buildLinux (args // rec {
  with (lib.kernel.whenHelpers version);
```

whoops, I forgot to add this part